lib/irb/completion.rb

```diff
@@ -58,19 +58,11 @@ def defined? do
 
     BASIC_WORD_BREAK_CHARACTERS = " \t\n`><=;|&{("
 
-    def self.absolute_path?(p) # TODO Remove this method after 2.6 EOL.
-      if File.respond_to?(:absolute_path?)
-        File.absolute_path?(p)
-      else
-        File.absolute_path(p) == p
-      end
-    end
-
```

Comment:
I think we can remove similar code in `lib/irb/ext/loader.rb`

https://github.com/ruby/irb/blob/bf20faa4e6a93edb07923522700deb56f1873618/lib/irb/ext/loader.rb#L27-L48

Reply:
Great catch 👍 I've removed it as well.